go/pkg/log/repository/log.go

```diff
@@ -144,7 +142,6 @@ func (r *LogRepository) UpdateCollectionCompactionOffsetPosition(ctx context.Con
 	if err != nil {
 		trace_log.Error("Error in updating record_compaction_offset_position in the collection table", zap.Error(err), zap.String("collectionId", collectionId))
 	}
-	trace_log.Info("Updated record_compaction_offset_position in the collection table", zap.Int64("offsetPosition", offsetPosition), zap.String("collectionId", collectionId))
```

Comment:
This one might be worth keeping for debugging.

Reply:
I picked it because having that offset in the past might be useful, as opposed to just logging guids like the other ones do.

Reply:
Discussed offline. Keeping this